Doc/library/profile.rst

```diff
@@ -132,7 +132,7 @@ the output by. This only applies when ``-o`` is not supplied.
 
 ``-m`` specifies that a module is being profiled instead of a script.
 
-   .. versionadded:: 3.7
+   .. versionadded:: 3.8
```

Comment:
here, you have modified the version for the '-m' flag for `cProfile`, not for `profile.py`, maybe (Doc/library/profile.rst) is not the right place.

Reply:
Maybe you should add a new section when we use `profile.py` via the command line.

Reply:
Good point, never make a change on a bus 😃 

profile.rst documents both `profile` and `cProfile` not sure what better place to add it.

Will add a new section to document the addition on `profile`, happy to move if you have a btter suggestion

Reply:
It's a good start if you want to create a new section for the `profile.py` tool.